scout/server/blueprints/variant/templates/variant/variant.html

```diff
@@ -586,27 +586,38 @@ <h4 class="modal-title">Cancel Sanger sequencing</h4>
           <a href="{{ gene.reactome_link }}" class="btn btn-outline-secondary" target="_blank">Reactome</a>
           <a href="{{ gene.expression_atlas_link }}" class="btn btn-outline-secondary" target="_blank">Expr. Atlas</a>
           <a href="{{ gene.clingen_link }}" class="btn btn-outline-secondary" target="_blank">ClinGen</a>
+          <!-- Display Alamut link only if bam is available -->
           {% if case.mt_bams or case.bam_files %}
-            <a href="{{ variant.alamut_link }}" class="btn btn-outline-secondary" target="_blank">Alamut</a>
+            {% set alamut_position = [ "GRCh38:" if case.genome_build == 38 ,  variant.chromosome, ":", variant.position, variant.reference, ">", variant.alternative]  %}
+            <form>
+            {% if variant.chromosome == "MT" and case.mt_bams %}
+              <input type="button" class="btn btn-outline-secondary" value='Alamut' onclick='httpGetLoadBam({{ alamut_position|join() }} ,
+                {{ case.mt_bams[0] }});' />
+            {% elif case.bam_files %} <!-- not a MT variant and bam file available -->
+            <input type="button" class="btn btn-outline-secondary" value='Alamut' onclick='httpGetLoadBam({{ alamut_position|join() }} ,
+              {{ case.bam_files[0] }});' />
```

Comment:
Ah, but that is the file name only? This should point to the endpoint for the scout bam file streaming url, right?